sklearn/multiclass.py

```diff
@@ -392,6 +394,87 @@ def intercept_(self):
         return np.array([e.intercept_.ravel() for e in self.estimators_])
 
 
+def _fit_regression(estimator, X, y):
+    estimator = clone(estimator)
+    estimator.fit(X, y)
+    return estimator
+
+
+def _parallel_helper(obj, methodname, *args, **kwargs):
+    """Private helper to workaround Python 2 pickle limitations"""
```

Comment:
Could you be more precise in your comment? Also add a unit test that checks that the object can be pickled.